ts/src/pro/cex.ts

```diff
@@ -1032,7 +1032,9 @@ export default class cex extends cexRest {
         const messageHash = 'orderbook:' + symbol;
         if (incrementalId !== storedOrderBook['nonce'] + 1) {
             delete client.subscriptions[messageHash];
-            client.reject (this.id + ' watchOrderBook() skipped a message', messageHash);
+            const error = new OperationFailed (this.id + ' watchOrderBook() skipped a message');
```

Comment:
@ttodua did you test if this works fine in c#?

Reply:
I've updated this PR and retested (i've also simulated incremental-id to be different, to cause throwing exceptions) and yes, it worked in c# too. 

just because of infrequent trades , it might timeout sometimes